components/brave_wallet/browser/eth_tx_manager.cc

```diff
@@ -691,7 +720,7 @@ void EthTxManager::MakeERC721TransferFromData(
   }
 
   const std::string chain_id =
-      json_rpc_service_->GetChainId(mojom::CoinType::ETH);
+      json_rpc_service_->GetChainIdSync(mojom::CoinType::ETH, absl::nullopt);
```

Comment:
Should Make* methods family get chain_id or origin from UI? or get active origin here? Not sure if this can be an issue